### DISCUSSION TITLE
Coursplay auf dedicated server

### DISCUSSION THREAD

**Discussion by Muf5in** [Courseplay/Courseplay_FS22#2207] [Q&A] [answered]
Hallo, weis jemand ob es möglich ist die neue Version von Coursplay auf einem Server zu benutzen? derzeit geht bei mir nur die 7.1.0.3
wenn ich versuche die 7.1.2.4 auf den Server zu laden und selber auch lade dann komme ich nicht auf den Server. 
Hat das schon jemand versucht?

**Comment by Tensuko** [comment, marked as answer]
Ja sicher geht das, nutzen wir selber.
Du darfst nur CP nicht ingame updaten, weil Giants nicht die Version prüft sondern das .zip file.
Lade von hier die .zip https://github.com/Courseplay/Courseplay_FS22/releases
Die packst du in deinen Mods Ordner und auf den Server, dann gehts.

**Reply by Muf5in** [reply]
Funktioniert bei mir nicht. Neuste Version ist auf dem Server und auf dem Rechner. Trotzdem wird beim Server Manager ein Ausrufezeichen angezeigt. Grund ist Coursplay.
![7 1 2 4](https://user-images.githubusercontent.com/121513204/209714191-2079655d-622d-421c-8335-2c6409be7874.png)
![Ausrufezeichen](https://user-images.githubusercontent.com/121513204/209714193-c7644686-182a-489c-a22a-5b0ebca75248.png)

**Reply by Muf5in** [reply]
Okay vielleicht hätte ich den Server mal neu starten sollen nachdem ich die Version geändert habe. Sorry mein Fehler

**Reply by Tensuko** [reply]
Ah okay, aber wenns jetzt läuft um so besser ;)
